UKEODHP/pulsar-demo
# feature/EODHP-12-get-started-with-pulsar

Created a CLI module demonstrating how to publish and subscribe to a Pulsar message queue.

To test on dev cluster:
1. Make sure argocd is pointing to feature/EODHP-12-get-started-with-pulsar branch.
2. Check Pulsar pods are all ready `k get pods -n pulsar`
3. Check proxy service is running `k get svc/pulsar-proxy -n pulsar`
4. Port forward pulsar-proxy to local machine `k port-forward -n pulsar svc/pulsar-proxy 6650:6650`
5. Run pulsar_demo module as indicated in README.md. Run at least one publish and subscribe simultaneously in separate terminals to make sure messages are sent and received.
